test/test.cpp

```diff
@@ -147,8 +147,10 @@ int main(int argc, char** argv)
         exit (EXIT_FAILURE);
     }
 
-    if (codec_str && strcmp (codec_str, "h265") == 0)
+    if (codec_str && strcmp (codec_str, "h265") == 0) {
       codec = VK_VIDEO_CODEC_OPERATION_DECODE_H265_BIT_EXT;
+      g_free (codec_str);
```

Comment:
move out this free out of the if, otherwise it won't be freed if it's different to "h265"